gulpfile.js

```diff
@@ -1,13 +1,22 @@
 const gulp = require('gulp');
 const connect = require('gulp-connect');
+const exec = require('child_process').exec;
+const sass = require('gulp-sass');
 
 gulp.task('hello', function() {
-	console.log("Hello World!");
+    console.log("Hello World!");
 });
 
 gulp.task('dev', function() {
-	connect.server({
-		root: 'src',
-		port: 9000
-	});
-});
\ No newline at end of file
+    exec('gulp sass');
```

Comment:
there is a better way in gulp to have dependent processes run.  Use that instead of exec